src/factory/client/clients.ts

```diff
@@ -91,19 +89,19 @@ export function createBtcClient(
   });
 
   // TODO: make sure Ledger works
-  if (accountType.includes('bitcoin_ledger')) {
-    const option = LEDGER_BITCOIN_OPTIONS.find((o) => o.name === accountType);
+  if (accountInfo.type.includes('bitcoin_ledger')) {
+    const option = LEDGER_BITCOIN_OPTIONS.find((o) => o.name === accountInfo.type);
     if (!option) {
-      throw new Error(`Account type ${accountType} not an option`);
+      throw new Error(`Account type ${accountInfo.type} not an option`);
     }
     const { addressType } = option;
     const ledgerProvider = new BitcoinLedgerProvider(
       {
         network: bitcoinNetwork,
         addressType,
-        baseDerivationPath,
-        basePublicKey: account?.publicKey,
-        baseChainCode: account?.chainCode,
+        baseDerivationPath: accountInfo.derivationPath,
+        basePublicKey: accountInfo?.publicKey,
+        baseChainCode: accountInfo?.chainCode,
         transportCreator: ledgerTransportCreator,
```

Comment:
Same story as other comment. Can't assume this transport will work on mobile

Reply:
fixed